trend_micro_vision_one_xdr/README.md

```diff
@@ -1,43 +1,67 @@
-# Agent Check: trend-micro-vision-one-xdr
-
 ## Overview
 
-This check monitors [trend-micro-vision-one-xdr][1].
+[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.
 
-## Setup
+This integration ingests the following logs:
 
-### Installation
+- **Workbench Alerts**: This endpoint contains information about all the standalone alerts triggered by detection models.
+- **Observed Attack Techniques**: This endpoint contains information about observed attack techniques from Detections, Endpoint Activity, Cloud Activity, Email Activity, Mobile Activity, Network Activity, Container Activity, and Identity Activity data sources.
 
-The trend-micro-vision-one-xdr check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+This integration collects logs from the sources listed above and sends them to Datadog for analysis with our Log Explorer and Cloud SIEM products
+* https://docs.datadoghq.com/logs/explorer/
```

Comment:
Can we make these hyperlinks on the words instead of direct links? [Log Explorer][3], [Cloud SIEM][4]

Reply:
Done